index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Welcome to the Peeps frontend</title>
+</head>
+<h1> Peeps </h1>
+  <body>
+    <label for="note-title">New user: username</label>
+    <input type="text" id="new-user-username" name="name" placeholder="eg Kelly123">
+    <br>
+    <label for="note-content">New user: password</label>
+    <input type="text" id="new-user-password" name="name" placeholder="Min 10 characters">
+    <br>
+    <button id="create-new-user">Create new user</button>
+    <br>
+    <br>
+    <label for="note-title">Existing user: username</label>
+    <input type="text" id="existing-user-username" name="name" placeholder="eg Kelly123">
+    <br>
+    <label for="note-content">Existing user: password</label>
+    <input type="text" id="existing-user-password" name="name" placeholder="Min 10 characters">
```

Comment:
I love your use of placeholders, will be including that in my projects in future